scripts/india_census/common/primary_abstract_data_variables.csv

```diff
@@ -0,0 +1,90 @@
+columnName,description,populationType,statType,measuredProperty,gender,age,socialCategory,literacyStatus,workerStatus,workerClassification,workCategory,workPeriod
+No_HH,Number of Households,Household,measuredValue,count,,,,,,,,
+TOT_P,Total Population,Person,measuredValue,count,,,,,,,,
+TOT_M,Total Males,Person,measuredValue,count,Male,,,,,,,
+TOT_F,Total Females,Person,measuredValue,count,Female,,,,,,,
+P_06,Population in the Age group 0-6,Person,measuredValue,count,,YearsUpto6,,,,,,
+M_06,Male Population in the Age group 0-6,Person,measuredValue,count,Male,YearsUpto6,,,,,,
+F_06,Female Population in the Age group 0-6,Person,measuredValue,count,Female,YearsUpto6,,,,,,
+P_SC,Population Scheduled Caste,Person,measuredValue,count,,,ScheduledCaste,,,,,
+M_SC,Male Population Scheduled Caste,Person,measuredValue,count,Male,,ScheduledCaste,,,,,
+F_SC,Female Population Scheduled Caste,Person,measuredValue,count,Female,,ScheduledCaste,,,,,
+P_ST,Population Scheduled Tribe,Person,measuredValue,count,,,ScheduledTribe,,,,,
+M_ST,Male Population Scheduled Tribe,Person,measuredValue,count,Male,,ScheduledTribe,,,,,
+F_ST,Female Population Scheduled Tribe,Person,measuredValue,count,Female,,ScheduledTribe,,,,,
+P_LIT,Literates,Person,measuredValue,count,,,,Literate,,,,
+M_LIT,Male literates,Person,measuredValue,count,Male,,,Literate,,,,
+F_LIT,Female literates,Person,measuredValue,count,Female,,,Literate,,,,
+P_ILL,Illiterates,Person,measuredValue,count,,,,Illiterate,,,,
+M_ILL,Male Illiterates,Person,measuredValue,count,Male,,,Illiterate,,,,
+F_ILL,Female Illiterates,Person,measuredValue,count,Female,,,Illiterate,,,,
+TOT_WORK_P,Total Workers,Person,measuredValue,count,,,,,Worker,,,
+TOT_WORK_M,Total Male Workers,Person,measuredValue,count,Male,,,,Worker,,,
+TOT_WORK_F,Total Female Workers,Person,measuredValue,count,Female,,,,Worker,,,
+MAINWORK_P,Main Workers,Person,measuredValue,count,,,,,Worker,MainWorker,,
+MAINWORK_M,Male Main Workers,Person,measuredValue,count,Male,,,,Worker,MainWorker,,
+MAINWORK_F,Female Main Workers,Person,measuredValue,count,Female,,,,Worker,MainWorker,,
+MAIN_CL_P,Main Cultivators,Person,measuredValue,count,,,,,Worker,MainWorker,Cultivators,
```

Comment:
Question whether this should be plural or singular?

@tjann

Reply:
I have converted into singular to keep it uniform. Let me know.

Reply:
Sorry, I mean things like: Cultivators, AgriculturalLabourers, HouseholdIndustries